app/templates/_labtemplates/_sass/style.scss

```diff
@@ -9,41 +9,41 @@
     ========================================================================= */
 
 @import 'settings/vars';
-@import '~supple/lib/settings/defaults';
-@import '~supple/lib/settings/responsive';
+@import 'node_modules/supple/lib/settings/defaults';
```

Comment:
supple-kit? Or are you using the "old" supple still?

Reply:
Old. This starterkit supports legacy browsers. We might drop that for 2.0 though.